src/main/kotlin/io/github/mojira/arisa/modules/CrashModule.kt

```diff
@@ -40,28 +43,64 @@ class CrashModule(
                 .toMutableList()
             textDocuments.add(TextDocument(body ?: "", created))
 
-            val infos = textDocuments
+            val crashes = textDocuments
+                .asSequence()
                 .filter(::isTextDocumentRecent)
-                .mapNotNull(::fetchInfo)
-            assertNotEmpty(infos).bind()
+                .map { processCrash(it) }
+                .filter { it.second.isRight() }
+                .map { extractCrash(it) }
+                .filter { it.second is Crash.Minecraft || it.second is Crash.Java }
+                .toList()
 
-            val mostRelevantInfo = infos.reduce(::getMoreRelevantInfo)
+            assertNotEmpty(crashes).bind()
 
-            if (mostRelevantInfo.modded) {
+            if (crashes.any { it.second is Crash.Minecraft && (it.second as Crash.Minecraft).modded }) {
```

Comment:
The current module prefers unmodded crashes, which makes sense, as one person might attach a modded crash, even though the mai report is about a vanilla crash.

Reply:
So how is the logic supposed to be for modded crashes?

1 crash report and its modded -> invalid
2 crash reports, one its modded -> scan that one
2 crash reports, both modded -> invalid?

Reply:
> 2 crash reports, one its modded -> scan that one

Not sure which one you meant with "scan that one", but the current module would scan the unmodded one in that case, which makes sense imo.

Reply:
the test for "should prefer crash that is not modded, if modded crash appears first" seems to be passing

Reply:
scan the other, yes

Reply:
Made it so it scans all crashes, but if there is no match and crash is modded to close as invalid

(all the detections we have happen whenever its modded or not, so its still useful to close them as duplicate instead of just invalid which would make the user create a new ticket just to get closed as duplicate)